build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
-    id 'fabric-loom' version '0.12-SNAPSHOT'
+    id 'fabric-loom' version '1.3-SNAPSHOT'
     id 'maven-publish'
-    id 'io.github.juuxel.loom-quiltflower' version '1.7.2'
+    id 'io.github.juuxel.loom-quiltflower' version '1.10.0'
```

Comment:
As of 1.11.0 this has been rebranded to vineflower, use that instead. 
https://github.com/Juuxel/loom-vineflower